### PR TITLE
feature/dummy data

### PR DESCRIPTION
**Pull request by paulmwatson**
- Clean up
- Dummy data structure

### CODE REVIEW

File: here_for_change/municipalities/templates/municipalities/ward_detail.html [hunk 1]
```diff
@@ -95,11 +95,11 @@ <h2 class="text-2xl md:text-3xl merriweather font-bold text-white text-center">
               <img class="rounded-lg w-20 h-20" src="https://assets.website-files.com/61a46f92a4845f1ed45ec01d/61d7e311bea0c7b4d41234e6_danny-europa.png"/>
               <div class="grow">
                 <div class="flex items-center justify-between">
-                  <p class="font-bold">Danny Europa</p>
+                  <p class="font-bold">{{content.councillor.name.value}}</p>
```
Comment (paulmwatson) [inline here_for_change/municipalities/templates/municipalities/ward_detail.html:98]: @Hazmid see how this pulls from the "dummy" data setup in the view below.

File: here_for_change/municipalities/views.py [hunk 2]
```diff
@@ -10,3 +11,37 @@ class WardDetail(DetailView):
     model = Ward
     slug_field = 'slug__iexact'
     slug_url_kwarg = 'slug'
+    page_content = {
+        'ward': {
+            'councillor': {
+                'name': {
+                    'value': 'Danny Europa',
```
Comment (paulmwatson) [inline here_for_change/municipalities/views.py:18]: @Hazmid so here is where `content.councillor.name.value` is set and pulled from.

File: here_for_change/municipalities/views.py [hunk 3]
```diff
@@ -10,3 +11,37 @@ class WardDetail(DetailView):
...
+                    'type': 'string',
+                    'updated': '2022/09/12 16:57'},
+                'political_party': {
+                    'value': 'African National Congress (ANC)',
+                    'type': 'string',
+                    'updated': '2021/02/12 05:57'},
+                'elected_date': {
+                    'value': datetime(2021, 1, 8),
```
Comment (paulmwatson) [inline here_for_change/municipalities/views.py:26]: @Hazmid you'll see some are values which need to be formatted in the template for the user. If you just emit `content.councillor.elected_date.value` directly you get `Jan. 8, 2021, midnight` but you'll need to use Django formatting in the template to display it as `8 January 2021`

File: here_for_change/municipalities/views.py [hunk 4]
```diff
@@ -10,3 +11,37 @@ class WardDetail(DetailView):
...
+                'contacts': {
+                    'email': {
+                        'primary': {
+                            'type': 'email',
+                            'value': 'info@capeagulhas.co.za',
+                            'updated': '2022/09/12 16:57',
+                            'feedback': {'positive': 24, 'negative': 24}}
+                    },
+                    'phone': {
+                        'value': '+27(0)284255500',
```
Comment (paulmwatson) [inline here_for_change/municipalities/views.py:38]: @Hazmid same for say this `phone` type, the data value needs formatting in the template.

File: here_for_change/municipalities/templates/municipalities/ward_detail.html (hunk 1 above)
Comment (Hazmid) [reply here_for_change/municipalities/templates/municipalities/ward_detail.html:98]: @paulmwatson  Fascinating. I noticed they were no imports. I like how clean Django is.

**Pull request merged**